sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -20,21 +20,22 @@
 # kernel.
 #
 # Arguments:
-# $1 == SDx's SDx bin directory where the real shell invocation that invokes
-#       the SDx compiler resides
-# $2 == path to the clang driver that's executing the shell script (e.g.
-#        your compiled Clang's build/bin)
-# $3 == The name of the source file without file extensions or temporary
-#       manglings, mostly used to generate some intermediate file names
-# $4 == The initial input file name, currently a .o file that's actually a .bc
-#       file in disguise
-# $5 == The systems temporary directory where we'll deposit some intermediate
-#       files
-# $6 == The name we wish to give our final output .xcl binary, important that it
+# $1  == SDx's SDx bin directory where the real shell invocation that invokes
+#        the SDx compiler resides
+# $2  == path to the clang driver that's executing the shell script (e.g.
+#         your compiled Clang's build/bin)
+# $3  == The name of the source file without file extensions or temporary
+#        manglings, mostly used to generate some intermediate file names
+# $4  == The initial input file name, currently a .o file that's actually a .bc
+#        file in disguise
+# $5  == The systems temporary directory where we'll deposit some intermediate
+#        files
+# $6  == The name we wish to give our final output .xcl binary, important that it
```

Comment:
Strange change in all the section.
There is 1 character added towards the start of each line.